zebra-consensus/src/block/subsidy/general.rs

```diff
@@ -25,20 +25,13 @@ pub fn halving_divisor(height: Height, network: &Network) -> Option<u64> {
         .activation_height(network)
         .expect("blossom activation height should be available");
 
-    // TODO: Add this as a field on `testnet::Parameters` instead of checking `disable_pow()`, this is 0 for Regtest in zcashd,
-    //       see <https://github.com/zcash/zcash/blob/master/src/chainparams.cpp#L640>
-    let slow_start_shift = if network.disable_pow() {
-        Height(0)
-    } else {
-        SLOW_START_SHIFT
-    };
-
-    if height < slow_start_shift {
+    if height < network.slow_start_shift() {
         unreachable!(
```

Comment:
Should we go with a panic instead?

```suggestion
        panic!(
```